### PR TITLE
feat: renamed cloud_account_id to have more meaningful name

### PR DESCRIPTION
**Pull request by pawelpaszki**
## Description
feat: renamed cloud_account_id to have mroe meaningful name

## Checklist (Definition of Done)

- [x] All acceptance criteria specified in JIRA have been completed
- [x] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
- ~~[ ] Documentation added for the feature~~
- [x] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- [ ] All PR comments are resolved either by addressing them or creating follow up tasks
- ~~[ ] Required metrics/dashboards/alerts have been added (or PR created).~~
- ~~[ ] Required Standard Operating Procedure (SOP) is added.~~
- ~~[ ] JIRA has been created for changes required on the client side~~

### CODE REVIEW

**Comment by pawelpaszki** [comment]
@pb82 @miguelsorianod

File: pkg/client/redhatsso/client_test.go [hunk 1]
```diff
@@ -800,7 +800,7 @@ func Test_rhSSOClient_UpdateServiceAccount(t *testing.T) {
 				Secret:      serviceAccount.Secret,
 				Name:        &name,
 				Description: &description,
-				OwnerId:     nil,
+				CreatedBy:   nil,
```
Comment (pb82) [inline pkg/client/redhatsso/client_test.go:803]: this change seems unrelated

**Review by pb82 — APPROVED** [review]

File: pkg/client/redhatsso/client_test.go (hunk 1 above)
Comment (pawelpaszki) [reply pkg/client/redhatsso/client_test.go:803]: yes - see https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1057
